lib/rules/time-min-milliseconds/index.js

```diff
@@ -118,27 +107,27 @@ const rule = (primary, secondaryOptions) => {
 		}
 
 		/**
-		 * @param {string} time
+		 * @param {import('postcss-value-parser').Dimension | {unit: null, number: null}} valueNode
 		 * @returns {boolean}
 		 */
-		function isAcceptableTime(time) {
-			const parsedTime = valueParser.unit(time);
+		function isAcceptableTime(valueNode) {
```

Comment:
[suggest] The name `dimension` seems more accurate:

```suggestion
		function isAcceptableTime(dimension) {
```